locations/storefinders/storefrontgateway.py

```diff
@@ -28,4 +30,11 @@ def parse(self, response):
             if location.get("openingHours"):
                 item["opening_hours"] = OpeningHours()
                 item["opening_hours"].add_ranges_from_string(location["openingHours"])
-            yield item
+            yield from self.post_process_item(item, response, location) or []
+
+    def pre_process_data(self, location, **kwargs):
+        """Override with any pre-processing on the item."""
+
+    def post_process_item(self, item, response, location):
+        """Override with any post-processing on the item."""
+        return item
```

Comment:
Make this `yield Item`, since the `post_process_item()` interface seems to have settled on iterables.

Reply:
https://github.com/alltheplaces/alltheplaces/pull/9452